rotkehlchen/db/dbhandler.py

```diff
@@ -1246,15 +1246,19 @@ def get_tokens_for_address(
             cursor: 'DBCursor',
             address: ChecksumEvmAddress,
             blockchain: SupportedBlockchain,
+            token_exceptions: set[ChecksumEvmAddress],
```

Comment:
I wonder ... would it make sense to turn this into a default argument with `None` as the default value?

Would need an extra if check at the end of thefunction but would make you stop needing to add this new `token_exceptions=set()` in many places.

Not sure tbh ... probably fine like that? What do you think?

Reply:
I believe it is better to leave as it is because the only place where I call it with empty set is... in tests. In the code I don't. Also I have changed it in task/assets.py ... there Initially I thought it was not needed but reading it now I believe it can introduce errors so adjusted it.
![image](https://github.com/user-attachments/assets/983889b2-91de-4a82-b701-dceccb0a6c32)